src/frontend/Komponenter/Personoversikt/Behandlingsoversikt.tsx

```diff
@@ -68,21 +60,19 @@ const lagTilbakekrevingslenke = (eksternFagsakId: number, behandlingId: string):
     return `${tilbakekrevingBaseUrl()}/fagsystem/EF/fagsak/${eksternFagsakId}/behandling/${behandlingId}`;
 };
 
-const Behandlingsoversikt: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
-    const [fagsak, settFagsak] = useState<Ressurs<Fagsak>>(byggTomRessurs());
-    const [tekniskOpphørFeilet, settTekniskOpphørFeilet] = useState<boolean>(false);
-    const [kanStarteRevurdering, settKanStarteRevurdering] = useState<boolean>(false);
-
-    const [visLagBehandlingModal, settVisLagBehandlingModal] = useState<boolean>(false);
+const Behandlingsoversikt: React.FC<{ fagsakPerson: IFagsakPerson }> = ({ fagsakPerson }) => {
+    const [fagsakOvergangsstønad, settFagsakOvergangsstønad] = useState<Ressurs<Fagsak>>(
+        byggTomRessurs()
+    );
+    const [fagsakBarnetilsyn, settFagsakBarnetilsyn] = useState<Ressurs<Fagsak>>(byggTomRessurs());
 
     const [feilFagsakHenting, settFeilFagsakHenting] = useState<string>();
-    const [tilbakekrevingBehandlinger, settTilbakekrevingbehandlinger] = useState<
-        Ressurs<TilbakekrevingBehandling[]>
-    >(byggTomRessurs());
+
     const { axiosRequest } = useApp();
-    const { toggles } = useToggles();
+    const { overgangsstønad: fagsakIdOvergangsstønad, barnetilsyn: fagsakIdBarnetilsyn } =
+        fagsakPerson;
 
-    const hentFagsak = () =>
+    const hentFagsak = (fagsakId: string, settFagsak: Dispatch<Ressurs<Fagsak>>) =>
```

Comment:
For oss "idioter": 
Får vi feilmelding noe sted hvis det ikke finnes fagsak for overgangsstønad eller barnetilsyn, eller blir det opprettet? 🤔

Reply:
Nei, blir ingen feilmelding bare tom side. Men man skal vel aldri komme til personoversikten (og dermed behandlingsoversikten) hvis man ikke har en fagsak hos oss?